ui/src/app/machines/components/MachineHeaderForms/AddChassis/AddChassisForm/AddChassisForm.tsx

```diff
@@ -0,0 +1,148 @@
+import { useEffect, useState } from "react";
```

Comment:
This file and `AddMachineForm` are basically unchanged from the move, apart from `resetOnSave` is now always true and `onSuccess` prop has been added.